src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -467,6 +464,14 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
       ResourceName="XA1023_dotnet"
       Condition="  '$(AndroidDexTool)' == 'dx' and '$(UsingAndroidNETSdk)' == 'true' "
   />
+  <AndroidWarning Code="XA1026"
+      ResourceName="XA1026"
+      Condition=" '$(AndroidUseAapt2)' == 'false' and '$(UsingAndroidNETSdk)' != 'true' "
```

Comment:
It's a pretty small thing, but for consistent handling of any funny invalid property values (like maybe `aapt2`), is it worth making these warning conditions exactly match the `Import` conditions: `'$(AndroidUseAapt2)' != 'true'` instead of `'$(AndroidUseAapt2)' == 'true'`?